### ISSUE TITLE
constructor (__init__) in a class

### ISSUE THREAD

**Issue by njzjz** [heavenshell/py-doq#83]
The docstring is added into `__init__`. But per [style guide of numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html),

> Use the same sections as outlined above (all except Returns are applicable). The constructor (__init__) should also be documented here, the Parameters section of the docstring details the constructor’s parameters.

They should be documented in the docstring of a class.

![image](https://user-images.githubusercontent.com/9496702/130314317-06c571f1-06fd-4e29-bcdc-96071b942891.png)

**Comment by heavenshell** [comment]
You can use your own template.

https://user-images.githubusercontent.com/56591/130319240-71f0db16-fa16-416a-97f5-473fac88e6e8.mp4

```
cat examples/noarg.txt 
"""{% if name == '__init__' %}Constructer{% else %}{{ name }}{% endif %}.
"""
```

**Comment by heavenshell** [comment]
oh, sorry I miss understood.

I'll add something like `ignore_inite` option.

**Comment by heavenshell** [comment]
Add ignore_init option.

Here is sample template file(class.txt)

```
"""{{ name }}.
{%- for def in defs %}
{%- if def.name == '__init__' %}

Parameters
----------
{% for p in def.params -%}
{{ p.argument }} :{% if p.annotation %} {{ p.annotation }}{% endif %}
    {{ p.argument }}
{% endfor %}
{% endif -%}
{% endfor -%}
"""

```

Run like following.

```python

class Foo:
    def __init__(self, arg1=None, arg2=None):
        pass

    def foo(self, arg1):
        pass


class Bar:
    def foo(self, arg1, arg2):
        pass


class Baz:
    class _Bar:
        def __init__(self, bar):
            pass

    def __init__(self, foo):
        pass
```

```console
cat sample.py | python -m doq.cli --ignore_init --template_path=./path/to/template
class Foo:
    """Foo.

    Parameters
    ----------
    arg1 :
        arg1
    arg2 :
        arg2

    """

    def __init__(self, arg1=None, arg2=None):
        pass

    def foo(self, arg1):
        """Summary of foo.

        Args:
            arg1
        """
        pass


class Bar:
    """Bar."""

    def foo(self, arg1, arg2):
        """Summary of foo.

        Args:
            arg1
            arg2
        """
        pass


class Baz:
    """Baz.

    Parameters
    ----------
    foo :
        foo

    """

    class _Bar:
        """_Bar.

        Parameters
        ----------
        bar :
            bar

        """

        def __init__(self, bar):
            pass

    def __init__(self, foo):
        pass
```

**Comment by heavenshell** [comment]
Released as 0.8.0
https://pypi.org/project/doq/0.8.0
